modules/fts/pages/fts-rebalance-failover.adoc

```diff
@@ -42,7 +42,7 @@ Checking the current value for `maxConcurrentPartitionMovesPerNode` in a cluster
 curl -XGET -u <username>:<password> http://<nodeIP>:8094/api/manager
 ----
 
-Note: When multiple partitions are built parallel, it needs more RAM and mandates a higher RAM quota. As the rebalance operations consume resources, it is always advisable to plan them during non-peak hours. One way to speed up the rebalance operation is to enable the movement of partitions in parallell in a configurable way.
+Note: When multiple partitions are built parallel, it needs more RAM and mandates a higher RAM quota. As the rebalance operations consume resources, it is always advisable to plan them during non-peak hours. One way to speed up the rebalance operation is to enable the movement of partitions in parallel in a configurable way.
```

Comment:
What does `.. in a configurable way` mean here.

Reply:
good spot! 
let me reword it!